build/kill_server.sh

```diff
@@ -13,7 +13,57 @@ echo "Project root: $PROJECT_ROOT"
 export PROJECT_ROOT
 echo "PROJECT_ROOT is set to: $PROJECT_ROOT"
 
+# Set fallback values for critical environment variables
```

Comment:
Add a command to load it from .env file instead of the below lines

# Set fallback values for critical environment variables
export CHAT_SERVICE_PORT=${CHAT_SERVICE_PORT:-8000}
export DB_USER=${DB_USER:-root}
export DB_PASSWORD=${DB_PASSWORD:-admin123}
export DB_HOST=${DB_HOST:-mysql_db}
export MYSQL_DATABASE=${MYSQL_DATABASE:-mydatabase}
export MYSQL_PORT=${MYSQL_PORT:-3306}
export MINIO_ROOT_USER=${MINIO_ROOT_USER:-minioadmin}
export MINIO_ROOT_PASSWORD=${MINIO_ROOT_PASSWORD:-minioadmin}
export MINIO_CONSOLE_PORT=${MINIO_CONSOLE_PORT:-9001}
export MINIO_SERVER_PORT=${MINIO_SERVER_PORT:-9000}
export WEAVIATE_PORT=${WEAVIATE_PORT:-8080}
export WEAVIATE_GRPC_PORT=${WEAVIATE_GRPC_PORT:-50051}
export OLLAMA_PORT=${OLLAMA_PORT:-11434}

Reply:
Added .env file loading to the kill_server.sh script.